misc/pytorch_toolkit/breast_ultrasound_simulation/src/exporter.py

```diff
@@ -0,0 +1,45 @@
+import torch
+from .model import GeneratorModel
+import argparse
+import os
+import subprocess
+
+def export_model_onnx(config_args):
+    device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+    print(torch.cuda.is_available())
+    model_path = config_args['model_path']
+    gen_old = GeneratorModel(1).to(device)
+    state_dict = torch.load(model_path)
+    gen_old.load_state_dict(state_dict["generator1_weights"])
+    onnx_model = config_args['onnx_model_name']
+    dummy_input = torch.randn(1,1,142,142).cuda()
+    torch.onnx.export(gen_old, dummy_input, onnx_model,
+                        opset_version = 11, do_constant_folding=True,
+                        input_names = ['input'], output_names=['output'],
+                        dynamic_axes={'input' : {0 : 'batch_size'}, 'output' : {0 : 'batch_size'}},
+                        verbose = False)
+
+def export_model_ir(config_args):
+    input_model = config_args['onnx_model_name']
+    input_shape = [1,1,142,142]
+    output_dir = os.path.split(config_args['ir_model_name'])[0]
+    export_command = f"""mo \
+        --framework onnx \
+        --input_model {input_model} \
+        --input_shape "{input_shape}" \
+        --output_dir {output_dir}"""
+    subprocess.run(export_command, shell = True, check = True)
+
+def main(config_args):
+    if config_args.onnx:
+        export_model_onnx(config_args)
+    if config_args.ir:
+        export_model_ir(config_args)
```

Comment:
I think, you should check that onnx model exists before converting it to the IR. If it does not, `export_model_onnx` should be called.